### ISSUE TITLE
fhir-ndr-et: builds break when the fhir schema changes

### ISSUE THREAD

**Issue by josephjclark** [OpenFn/adaptors#776]
tl;dr: don't download the main.zip, but version lock the implementation spec version we download

The fhir-ndr-et adaptor (the jembi one) uses an automated build process.

What it does is, whenever the build runs in CI, it downloads the LATEST fhir schema from the implementation. And from this schema it generates new source files.

And if the schema has changed, the source  will change.

And if there is a diff in the source, the publish step will fail.

What this all means that when the jembi spec is updated, all our releases are BLOCKED because the publish step will fail.

Options

* Version-lock the schema. Right now we just download the master file, whatever it happens to be. If we can build to a version, that would make a lot more sense! We should also declare somewhere in docs what jembi schema version is being used (maybe even provide an API for it in job code). This is the answer IF we can get a versioned bundle from the implementation.
* Disable the build entirely. But actually you'll still get a git diff  on build, because the source files for the adaptor don't exist. So it'll look like the adaptor files have been deleted. And actually THIS is really bad because it might result in the adaptor being published without any source files (it's probably harmless in practice because there's no version bump)
* Check the schema bundle into github. Not really keen on this.

**Comment by josephjclark** [comment]
I think the problems with versioning this are:
1) I'm not sure that the implementation guide from which I'm downloading the spec is versioned. Hoping to find a magic URL which will allow me to get previous versions but I haven't found it yet (I have found a bunch of broken links however)
2) Even if I can get a versioned spec, I'm not sure I trust it. Something has clearly changed in the implementation guide in the last couple of weeks, but the version number in the spec seems to be 1.0.1 (on both versions). Maybe I'm not interpreting the version correctly, but also maybe jembi don't expect anyone to rely on the version number and are making informal updates.

Between the two things I'm sure how to control the source spec without a) checking it in or b) disabling the build.

**Comment by josephjclark** [comment]
This is actually really easy!

All the generated code is checked in, so there's no need for the build to run in CI.

So we take build:src out of the build step and make that a manual, offline process. 

We update docs and we we make a build:clean or build:update which deletes the spec.json file
